tests/conftest.py

```diff
@@ -66,18 +66,26 @@ def limited_rate():
     def unlimited_rate():
         return 'test'
 
-    Limiter(
-            app_,
-            key_func=obj_or_import_string(
-                app_.config.get('RATELIMIT_KEY_FUNC'),
-                default=useragent_and_ip_limit_key)
-        )
     return app_
 
 
+@pytest.fixture()
+def app_with_no_limiter(base_app):
+    """Flask application fixture without limiter registered."""
+    with base_app.app_context():
+        yield base_app
+
+
 @pytest.yield_fixture()
 def app(base_app):
     """Flask application fixture."""
+    base_app.config['USE_FLASK_LIMITER'] = True
```

Comment:
This config variable is not needed anymore and you can instead pass ```RATELIMIT_APPLICATION=wrap_rate_limit```.
Also, it would now make sense to transfer the logic regarding the Limiter from the base_app to app.